### DISCUSSION TITLE
Oh My CatBlock - 可能是这个星球上最可爱的ClipCC扩展

### DISCUSSION THREAD

**Discussion by SimonShiki** [Clipteam/clipcc-gui#53] [扩展发布/Extensions sharing]
扩展英文名称：Oh My CatBlocks
扩展中文名称：并没有
扩展介绍：喵喵喵喵喵喵喵喵喵喵（翻译：如图所示）
![CatBlocks](https://user-images.githubusercontent.com/59327348/129716375-5fd3aa8f-0e11-427b-bd4e-4a21a819ef48.png)
开源地址：[点这里](https://github.com/SinanGentoo/oh-my-catblocks)
下载地址：[点这里](https://github.com/SinanGentoo/oh-my-catblocks/releases)

**Comment by SimonShiki** [comment]
已适配最新版本））

**Reply by JasonXu134590** [reply]
orz
